### PR TITLE
add recipe: shandy

### PR DESCRIPTION
**Pull request by alanpaulprice**

### CODE REVIEW

File: src/recipes/shandy.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Shandy",
+    "description": "This makes for a luscious beer with a hint of lemonade.",
+    "github": "<Your Github username>",
```
Comment (alfg) [inline src/recipes/shandy.json]:
```suggestion
    "github": "alanpaulprice",
```

File: src/recipes/shandy.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        }
+    ],
+    "directions": [
+        "Pour the beer into a pint glass.",
+        "Slowly add the lemonade.",
+        " Stir gently."
+    ],
+    "image": "shandy.jpg",
+    "source": "https://www.allrecipes.com/recipe/149943/shandy/",
+    "keywords": []
```
Comment (alfg) [inline src/recipes/shandy.json]: Any keywords to add?

Comment (alanpaulprice) [reply src/recipes/shandy.json]: Done!

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**